content/post/contributors/yokwejuste.md

```diff
@@ -0,0 +1,12 @@
+#Hey I'm yokwejuste 😁
+***Best slogan:*** : Beauty better than ugly.
+
+I'm a Computer engineering student at University of Bamenda...
```

Comment:
> ...

Why?

Reply:
@yokwejuste can you remove the three dots and replace them by a single one ?